internal/app/init/pkg/system/services/kubelet.go

```diff
@@ -87,11 +105,12 @@ func (k *Kubelet) Start(data *userdata.UserData) error {
 		{Type: "cgroup", Destination: "/sys/fs/cgroup", Options: []string{"rbind", "rshared", "rw"}},
 		{Type: "bind", Destination: "/dev", Source: "/dev", Options: []string{"rbind", "rshared", "rw"}},
 		{Type: "bind", Destination: "/var/run", Source: "/run", Options: []string{"rbind", "rshared", "rw"}},
-		{Type: "bind", Destination: "/var/lib/kubelet", Source: "/var/lib/kubelet", Options: []string{"rbind", "rshared", "rw"}},
-		{Type: "bind", Destination: "/var/lib/containerd", Source: "/var/lib/containerd", Options: []string{"rbind", "rshared", "rw"}},
+		// 'blind' Support for calico / containerd / kubelet / rook(?)
+		//  or should we be very explicit here?
+		{Type: "bind", Destination: "/var/lib", Source: "/var/lib", Options: []string{"rbind", "rshared", "rw"}},
 		{Type: "bind", Destination: "/var/log/pods", Source: "/var/log/pods", Options: []string{"rbind", "rshared", "rw"}},
-		{Type: "bind", Destination: "/etc/kubernetes", Source: "/etc/kubernetes", Options: []string{"bind", "rw"}},
-		{Type: "bind", Destination: "/etc/os-release", Source: "/etc/os-release", Options: []string{"bind", "ro"}},
+		{Type: "bind", Destination: "/etc/kubernetes", Source: "/var/etc/kubernetes", Options: []string{"bind", "rw"}},
```

Comment:
I originally had `/var/etc/kubernetes` explicitly here and ended up changing the `/etc/kubernetes` because it was one less thing to think about when setting up a service. Also if we move away from `/var/etc/kubernetes` and end up making `/etc/kubernetes` writable in some other way (another partition, or different symlink) then we don't have to worry about it here.